src/Session/Session.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Pantheon\Terminus\Session;
+
+use Terminus\Caches\FileCache;
+use Terminus\Models\User;
+
+class Session {
+  /**
+   * @var FileCache
+   */
+  protected $cache;
+
+  /**
+   * @var object
+   */
+  protected $data;
+
+  /**
+   * Instantiates object, sets session data
+   */
+  public function __construct($fileCache) {
+    $this->cache = $fileCache;
+    $session     = $this->cache->getData('session');
+    $this->data  = $session;
+    if (empty($session)) {
+      $this->data = new \stdClass();
```

Comment:
I dunno. I think this should be an array probably but this thing was a pretty lazy copy from the old version. It'll need to be refactored once `Auth()` is refactored to use it. For now I say leave this be.